var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -111,6 +111,8 @@ class Trilinos(CMakePackage, CudaPackage):
     variant('minitensor',   default=False, description='Compile with MiniTensor')
     variant('muelu',        default=True, description='Compile with Muelu')
     variant('nox',          default=False, description='Compile with NOX')
+    variant('panzer',       default=False, description='Compile with Panzer')
+    variant('panzerdofmgr', default=True, description='Compile with PanzerDofMgr')
```

Comment:
@ikalash, could you take out the commits that change the `trilinos` package, since we don't plan to use it?  I think this will make rebasing easier in the future.

Reply:
Yes, I'll do this.

Reply:
Done.